src/core/request.js

```diff
@@ -138,9 +156,9 @@ export default function request(options) {
             return Promise.resolve();
         };
 
-        return computeHeaders().then(function(customHeaders) {
-            return new Promise(function(resolve, reject) {
-                var noop;
+        return computeHeaders().then(customHeaders => {
+            return new Promise((resolve, reject) => {
+                const noop = void(0);
```

Comment:
Or `void 0` or inline comment for the object property below for eslint to not consider the value undefined while checkup. My opinion is that in rare cases it is more straightforward to have the original undefined value with eslint ignore comment then tricking it with void.

Reply:
Good idea, I went with `undefined` +  inline comment.